webroot/src/pages/SelectPrivileges/SelectPrivileges.ts

```diff
@@ -89,6 +83,10 @@ export default class SelectPrivileges extends mixins(MixinForm) {
         return name;
     }
 
+    get isPhone() {
+        return this.$matches.phone.only;
+    }
+
     get privilegeList(): Array<License> {
```

Comment:
Not sure if this is the correct spot, but should be sort the UI list alphabetically?

<img width="339" alt="Screenshot 2024-11-20 at 10 58 55 AM" src="https://github.com/user-attachments/assets/249d9df8-992e-473a-b58f-bab9abcda9b5">